templates/admin/admin_index.html

```diff
@@ -0,0 +1,6 @@
+{% extends "admin/index.html" %}
+{% load i18n %}
+
+{% block footer %}
+<div id="footer">{% trans "Version" %}: {{ version }}</div>
```

Comment:
Would it be clearer to label this "git hash" or something like that? If the VERSION is changed to proper version number some day it could be then changed back?

Reply:
This project doesn't have an opinion about what the user of this software might put into the `VERSION` environment variable, where `settings.VERSION` is read from. Although `settings.VERSION` defaults to the output of a `git describe` command (but that's not a git hash either). So in that sense I would keep this as it is.

Or then make the whole string configurable, so one can input anything they want into it. But that sounds quite an overkill.

Reply:
I'm just saying that it would be nice to know exactly what that is. As it will be the result of git describe, could it say that it's a result of `git describe` somewhere? But maybe it's just something our team should know or maybe it should be documented in Confluence.

Reply:
The `git describe` is a backup plan (and IMO could even be removed from the code). The main way to use this is to set the `VERSION` environment variable. But within this code base we can't control what value anybody decides to put in that environment variable.